src/test/java/com/google/events/cloud/pubsub/v1/MessagePublishedDataTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.events.cloud.pubsub.v1;
+import static org.junit.Assert.assertEquals;
+
+import com.google.gson.Gson;
+import org.junit.Test;
+
+import java.util.Base64;
+
+public class MessagePublishedDataTest {
+  private String decode(String data) {
+    return new String(Base64.getDecoder().decode(data));
```

Comment:
This uses the platform-default character encoding. It would be better to use a specific encoding, even if we're expecting it to always be ASCII data, e.g. passing in `StandardCharsets.UTF_8` as a second argument.